README.md

```diff
@@ -10,10 +10,113 @@ This is particularly useful when provisioning VMs with vagrant.
 
 ## WARNING
 
-This is pretty much just a prototype at the moment. Its functionality is fairly
-limited and it's almost certainly not ready for production use.
+This is pretty much just a prototype at the moment. Its functionality is quite
+limited and it hasn't been tested on a wide variety of environments. It meets
+our fairly needs, but we wouldn't recommend using it in production without
```

Comment:
Thanks. I edited that sentence a bunch of times and must have lost the word somewhere along the way.